synology/start_tunnel_zabbix_proxy_to_server.sh

```diff
@@ -13,6 +14,13 @@ if [ -z "${tmp_dir}" ]; then
   tmp_dir=$(mktemp -d)
 fi
 
+known_hosts_file_server="${tmp_dir}/known_hosts"
+
+cat <<EOF > "${known_hosts_file_server}"
+${localhost}       ${server_key}
+[${localhost}]:7105  ${server_key}
```

Comment:
This is not going to work, this server will have a different key, you need the actual public host key which you can find in `/etc/ssh/ssh_host_ed25519_key.pub` (make sure to only put the **public** one on github!!).